src/Collections-Strings/String.class.st

```diff
@@ -1048,6 +1048,14 @@ String >> endsWithAColon [
 
 ]
 
+{ #category : #comparing }
+String >> endsWithAnyOf: aCollection [
+	aCollection do:[:suffix|
+		(self endsWith: suffix) ifTrue:[^true].
+	].
+	^false
+]
```

Comment:
I think this addition was not intended.

Reply:
This was removed in another PR. Iceberg now added it again ... mmh.

Reply:
it is likely just shown wrong in the diff tool of GitHub